bin/hr/hr.go

```diff
@@ -351,6 +366,7 @@ func main() {
 	pflag.BoolVar(&conv.formatter.TinyFormat, "tiny", false, "use penlog hr-tiny format")
 	pflag.StringVarP(&prioLevelRaw, "priority", "p", "debug", "show messages with a lower priority level")
 	pflag.StringArrayVarP(&filterSpecs, "filter", "f", []string{}, "write logs to a file with filters")
+	pflag.BoolVar(&conv.showProgress, "show-progress", false, "replace the consecutive displayment of info messages with a progress bar")
```

Comment:
```suggestion
	pflag.BoolVar(&conv.showProgress, "progress", false, "replace the consecutive displayment of info messages with a progress bar")
```

Reply:
Hmm. Not sure if this naming is correct. It is not a progress bar. It is more a `--collapse` or something.

Reply:
I stcked to @peckto's naming but I agree

Reply:
I know, I don't like my `--collapse` suggestion very well. But it fits a bit more to the actual thing.

Reply:
In the new version, it's called volatile-infos, maybe that's a better fititng name.

Reply:
last complain: what about `--volatile` (without the `-info`)?